docs/basics/getting-started.md

```diff
@@ -2,10 +2,15 @@
 
 Welcome to the Self Hosting basics section of the wiki. Here you will find a bunch of links and useful information for you to get started on your own Self-Hosting journey.
 
+### Content on the Self Hosted Wiki is available under [GNU Free Documentation License 1.3.](http://www.gnu.org/licenses/fdl-1.3.html)
```

Comment:
Actually, it's not, it's regular GPL 3

https://github.com/selfhostedshow/wiki/blob/dev/LICENSE